packages-ts/gauntlet-terra-contracts/src/commands/tooling/updateContractAdmin.ts

```diff
@@ -0,0 +1,80 @@
+import { Result } from '@chainlink/gauntlet-core'
+import { logger, prompt } from '@chainlink/gauntlet-core/dist/utils'
+import { TerraCommand, TransactionResponse } from '@chainlink/gauntlet-terra'
+import { AccAddress, MsgUpdateContractAdmin } from '@terra-money/terra.js'
+import { CATEGORIES } from '../../lib/constants'
+
+type CommandInput = {
+  newAdmin: string
+  contract: string
+  forceNewAdmin?: boolean
+}
+
+export default class UpdateContractAdmin extends TerraCommand {
+  input: CommandInput
+
+  static description = 'Updates contract admin'
+  static examples = [
+    `yarn gauntlet tooling:transfer_contract_admin --network=bombay-testnet --to=<NEW_ADMIN> <CONTRACT_ADDRESS>`,
+  ]
+
+  static id = 'tooling:transfer_contract_admin'
```

Comment:
Consider renaming as `'tooling:update_contract_admin'` as that is explicitly how the message is named. `transfer` is not a key word here in this context.

Additionally, the `tooling:` namespace could be split into multiple and this CMD could have an id like `contract:update_admin`, accompanying the `contract:migrate` & `contract:migrate_code` CMDs.